source/core/slang-io.cpp

```diff
@@ -286,7 +286,23 @@ namespace Slang
             // If ioBuilder doesn't end in a delimiter, add one
             if (!isDelimiter(ioBuilder[ioBuilder.getLength() - 1]))
             {
-                ioBuilder.append(kPathDelimiter);
```

Comment:
Perhaps we should have a function that given a builder (and/or a slice) returns the delimiter type. Then we add a parameter to `append` that takes the delimiter, with the default being the usual.

That would move the logic of determining the used delimiter outside of this function, such that it is used when it's needed - which I'm assuming (?) is not common.

Reply:
We can clean this up. For now I am going to merge this PR since it fixes a language server bug that impacts many users.

Reply:
Ok no worries.